Dockerfile

```
@@ -9,11 +9,15 @@ FROM debian:buster AS build
 # Allow users to specify forked node, or specific commit
 # If not fallback to release branch (or alternatively just specify branch)
 ARG NODE_REPO=https://github.com/casper-network/casper-node.git
-ARG NODE_COMMIT=
 ARG CLIENT_REPO=https://github.com/casper-ecosystem/casper-client-rs.git
-ARG CLIENT_COMMIT=
-ARG NODE_GITBRANCH=release-1.5.6
+ARG SIDECAR_REPO=https://github.com/casper-network/casper-sidecar.git
+
+ARG NODE_COMMIT=
+ARG NODE_GITBRANCH=feat-2.0
```

I think it should be pointed to Casper Node v2.0-rc3 (latest locked release), as `feat-2.0` is still under development:

```suggestion
ARG NODE_GITBRANCH=release-2.0.0-rc3
```